data/eurostat/projection.json

```diff
@@ -0,0 +1,65 @@
+{
+    "keep": [
```

Comment:
Set of columns to keep is pretty small: Maybe just integrate it into script?

Reply:
This seems good to me. Not a fan of long lists in code if I can't help it, and this is just data.